ports/portaudio/vcpkg.json

```
@@ -1,8 +1,8 @@
 {
   "name": "portaudio",
-  "version-string": "2020-02-02",
-  "port-version": 2,
+  "version-string": "2020-02-20",
+  "port-version": 3,
```

```suggestion
  "port-version": 0,
```

Port version should reset to 0 when the main version changes.

---

After I moved this to zero, I needed to remove the line all together to make it pass the "./vcpkg format-manifest --all"